### PR TITLE
docs for use-bundled-mocha-for-reporters

### PR DESCRIPTION
**Pull request by Bkucera**
docs for https://github.com/cypress-io/cypress/pull/6951

### CODE REVIEW

**Comment by jennifer-shehane** [comment]
@Bkucera I believe some of the examples may need updating because they show installing mocha. Not totally sure? Some places showing install of `mocha` with reporters:

- https://docs.cypress.io/guides/tooling/reporters.html#Spec-to-STDOUT-save-JUnit-XML-files
- https://docs.cypress.io/guides/tooling/reporters.html#Spec-to-STDOUT-produce-a-combined-Mochawesome-JSON-file

File: source/guides/tooling/reporters.md [hunk 1]
```diff
@@ -19,8 +19,6 @@ Once you've read through the documentation below, we invite you to experience th
 
 # Custom reporter
 
-You need to install any peer dependencies the reporter requires, even if they're bundled with Cypress. For example, {% url "mochawesome" https://github.com/adamgruber/mochawesome %} requires `mocha` as a peer dependency. You will need to install `mocha` as a dev dependency of your own project for it to work.
-
```
Comment (Bkucera) [inline source/guides/tooling/reporters.md:23]: put in warning block, mention pre-4.4.1

Comment (jennifer-shehane) [reply source/guides/tooling/reporters.md:23]: We don't mention previous version support in our docs - they should be up to date with current version. 

If you want to note changes, you can add a History table at the bottom to reflect version changes.

**Comment by jennifer-shehane** [comment]
As I mentioned above, 

>We don't mention previous version support in our docs - they should be up to date with current version.
>
>If you want to note changes, you can add a History table at the bottom to reflect version changes.

Adding notes everywhere in our docs about doing this in x version and this support in x version is opening a can of worms that we con't intend to support or have a process around.

I'll be removing the note about previous versions.

File: source/guides/tooling/reporters.md (hunk 1 above)
Comment (Bkucera) [reply source/guides/tooling/reporters.md:23]: :+1:

**Pull request merged**